tests/test_map_citations_murs.py

```diff
@@ -0,0 +1,164 @@
+            us_code_citations=[
+                # re-mapped citations
+                dict(text="52 U.S.C. 30101", url="http://api.fdsys.gov/link?collection=uscode&link-type=html&title=52&year=mostrecent&section=30101"),
+                dict(text="52 U.S.C. 30102", url="http://api.fdsys.gov/link?collection=uscode&link-type=html&title=52&year=mostrecent&section=30102"),
+            ],
+            regulations_citations=regulations_citations,
+            id='mur1'
+        )
+
+
+        mur2 = self.mur_factory(
+            us_code_citations=[
+                dict(text="2 U.S.C. 433", url="http://api.fdsys.gov/link?collection=uscode&link-type=html&title=2&year=mostrecent&section=433"),
+                dict(text="2 U.S.C. 434", url="http://api.fdsys.gov/link?collection=uscode&link-type=html&title=2&year=mostrecent&section=434"),
+            ],
+            regulations_citations=regulations_citations,
+            id='mur2'
+        )
+        expected_mur2 = self.mur_factory(
+            us_code_citations=[
+                # re-mapped citations
+                dict(text="52 U.S.C. 30103", url="http://api.fdsys.gov/link?collection=uscode&link-type=html&title=52&year=mostrecent&section=30103"),
+                dict(text="52 U.S.C. 30104", url="http://api.fdsys.gov/link?collection=uscode&link-type=html&title=52&year=mostrecent&section=30104"),
+            ],
+            regulations_citations=regulations_citations,
+            id='mur2'
+        )
+
+        expected_bulk_ops = [dict(_op_type='update', _id='mur1', doc=expected_mur1), dict(_op_type='update', _id='mur2', doc=expected_mur2)]
+
+        mock_scan.return_value = (self.dict_to_es_result(mur1), self.dict_to_es_result(mur2))
+        mock_bulk.return_value = [2, []] # Update count, error list
+
+        load_legal_docs.remap_archived_murs_citations()
+
+        # Assert no real calls to ES
+        es_mock.assert_not_called()
+
+        # Scan was called exactly once
+        assert len(mock_scan.mock_calls) == 1
+
+        # Assert bulk was called exactly once
+        assert len(mock_bulk.mock_calls) == 1
+
+        # Check bulk was called with the re-mapped citations
+        args, kwargs = mock_bulk.call_args
+        _, bulk_ops = args
+        assert list(bulk_ops) == expected_bulk_ops
+
+
+class TestGetCitations(unittest.TestCase):
+
+    def test_reclassify_pre2012_citation(self):
+        # spot check a few cases from the csv
+        assert reclassify_statutory_citation.reclassify_pre2012_citation('2', '431') == ('52', '30101')
+        assert reclassify_statutory_citation.reclassify_pre2012_citation('2', '437g') == ('52', '30109')
+        assert reclassify_statutory_citation.reclassify_pre2012_citation('42', '1973aa-1a') == ('52', '10503')
+
+        # and a fallback
+        assert reclassify_statutory_citation.reclassify_pre2012_citation('99', '12345') == ('99', '12345')
+
+
+    @mock.patch.object(reclassify_statutory_citation, 'reclassify_pre2012_citation')
```

Comment:
I'm a fan of testing single components at a time, so this test is for testing `get_citations` and assumes `reclassify_pre2012_citation` is tested elsewhere (it is). The tests below are at a "higher level", but simpler in the test coverage. There's also another test that does "the whole thing" by running the `load_archived_murs` job.

The purpose of the mock is to test that `get_citations` parses the title and section properly and then constructs the citation text and url properly. Overall this approach has these advantages
- it's clear to see inputs and outputs
- you don't have to think about (or debug) what `reclassify_pre2012_citation` is doing
- it's easy to add corner cases to this test